core/domain/exp_fetchers_test.py

```diff
@@ -324,207 +323,3 @@ def test_does_not_convert_up_to_date_exploration(self):
             exploration.states_schema_version,
             feconf.CURRENT_STATE_SCHEMA_VERSION)
         self.assertEqual(exploration.to_yaml(), self._up_to_date_yaml)
-
-    def test_migration_then_reversion_maintains_valid_exploration(self):
```

Comment:
Is the revert functionality tested elsewhere? If not, probably we should keep this test? The job run can be replaced by calling the migration function itself, right?

Reply:
Done, returned the test back.